client/lib/gsuite/has-gsuite-with-another-provider.js

```diff
@@ -7,7 +7,7 @@ import { get } from 'lodash';
  * @returns {boolean} - true if the domain is with another provider, false otherwise
  */
 export function hasGSuiteWithAnotherProvider( domain ) {
-	const domainStatus = get( domain, 'googleAppsSubscription.status', '' );
+	const domainStatus = getGSuiteSubscriptionStatus( domain );
```

Comment:
Could we please rename that variable which has a wrong name? I've wanted to do that for so long 😀.
```suggestion
	const status = getGSuiteSubscriptionStatus( domain );
```